src/main/java/org/example/homework4/LeapYear.java

```diff
@@ -0,0 +1,16 @@
+package org.example.homework4;
+
+public class LeapYear {
+    public static void main(String[] args) {
+        int year = 2024;
+        if (isLeapYear(year)) {
+            System.out.println(year + " is a Leap Year");
+        } else System.out.println(year + " isn't a Leap Year or out of range [1;9999]");
```

Comment:
There is no chek for 1;9999 interval validation, please fix

Reply:
Can you please explain what do i need to write here, cant figure it out

Reply:
You don't check in the code that year is >= 1 and <= 9999